.env

```diff
@@ -98,8 +98,8 @@ ESPRESSO_BUILDER_ETH_MNEMONIC=${ESPRESSO_SEQUENCER_ETH_MNEMONIC}
 ESPRESSO_BUILDER_SERVER_PORT=41003
 ESPRESSO_BUILDER_CHANNEL_CAPACITY=1024
 ESPRESSO_BUILDER_BOOTSTRAPPED_VIEW=0
-ESPRESSO_BUILDER_WEBSERVER_RESPONSE_TIMEOUT_DURATION=1s
-ESPRESSO_BUILDER_BUFFER_VIEW_NUM_COUNT=15
+ESPRESSO_BUILDER_WEBSERVER_RESPONSE_TIMEOUT_DURATION=1500ms
```

Comment:
@jbearer please check this!

Reply:
Seems reasonable. With `ESPRESSO_ORCHESTRATOR_BUILDER_TIMEOUT=2s` this means we can take the maximum server reponse time to process a request and still have 500ms for the response to propagate to the leader before they give up